db/migrate/20160629210827_add_upvote_count_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUpvoteCountToComments < ActiveRecord::Migration
+  def change
+    add_column :comments, :upvote_count, :integer, default: 0
+  end
+end
```

Comment:
you can add multiple columns in one migration. I wouldn't suggest changing these, but in the future, you can do something like:
```ruby
class AddVoteCountsToTables < AR::Migration
  add_column :posts, :upvote_count, :integer, default: 0
  add_column :posts, :downvote_count, :integer, default: 0
  add_column :comments, :upvote_count, :integer, default: 0
  add_column :comments, :downvote_count, :integer, default: 0
end
```